### PR TITLE
Added Baselines

### PR DESCRIPTION
**Pull request by Henry-WattTime**
From Issue #117

### CODE REVIEW

File: Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md [hunk 1]
```diff
@@ -208,8 +208,20 @@ You MUST include an estimate of all the embodied emissions for the hardware used
...
+- Benchmark 
+- User
+- Machine
+- Minute/time unit
+- Device
+- Physical site
+- Data volume
+- Job 
+- Transaction
+- Mining activity
```
Comment (Sealjay) [inline Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md:223]:
```suggestion
- Distributed Ledger/Blockchain mining activity
```

File: Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md [hunk 2]
```diff
@@ -208,8 +208,20 @@ You MUST include an estimate of all the embodied emissions for the hardware used
...
-### Preset List for Baselines
-[placeholder]
+### Preset List for Baselines (R)
+Acceptable baselines include:
+- API call/request 
+- Benchmark 
+- User
+- Machine
+- Minute/time unit
+- Device
```
Comment (Sealjay) [inline Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md:218]: Does this (and other similar items like physical site, or data volume) need some form of sizing? I can clearly see how User, or API call relates to R. But physical site, data volume, and device, seem like they could have significant variance.

Comment (jawache) [reply Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md:218]: - I think size is fine, like GB, MB? Or are you saying we need to list every term that can be used to define the size of a data?

- Device I think makes sense if you are perhaps an IoT cloud service and you scale by the number of devices that are sending you data?

Comment (atg-abhishek) [reply Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md:218]: We should definitely have some sizing suggestions, and the fact that we if reword it as "suggested" as suggested (pun intended! ;) ) by @jawache then that would make it clearer.

**Pull request merged**